patroni/postgresql/bootstrap.py

```diff
@@ -400,9 +400,9 @@ def post_bootstrap(self, config: Dict[str, Any], task: CriticalTask) -> Optional
 END;$$""".format(f, quote_ident(rewind['username'], postgresql.connection()))
                         postgresql.query(sql)
 
-                for name, value in (config.get('users') or {}).items():
-                    if all(name != a.get('username') for a in (superuser, replication, rewind)):
-                        self.create_or_update_role(name, value.get('password'), value.get('options', []))
+                if config.get('users'):
```

Comment:
When the feature is removed we probably better complain only if there are any users different from superuser, replication, rewind:
```suggestion
                if any(user not in (superuser, replication, rewind) for user in (config.get('users') or {}).keys()):
```

Reply:
We still need to prevent some users from initializing those three users using the 'users' option. They might provide other initialization options, which will be ignored after we have removed this feature.

Reply:
For example:
replication:
    options:
          - createdb (which will be ignored but not complained)

Reply:
These three users are already created earlier.  I am not suggesting to create them once again (it didn't happen before). My suggestion it to show not error if there are no other users.

Also, probably I am overthinking it.

Reply:
got you, let's complain everything about deleted "users" options when people tries to use it.